flow/Stats.actor.cpp

```diff
@@ -45,21 +45,32 @@ double Counter::getRate() const {
 }
 
 double Counter::getRoughness() const {
-	double elapsed = now() - interval_start;
+	double elapsed = now() - roughness_interval_start;
 	if(elapsed == 0) {
 		return 0;
 	}
 
+	// If we have time samples t in T, and let:
+	// n = size(T) = interval_delta
+	// m = mean(T) = elapsed / interval_delta
+	// v = sum(t^2) for t in T = interval_sq_time
```

Comment:
I'm still not sure what `t` is defined. 
Do you have a very simple example about the `n, m, v`?

Reply:
`T` is the set of samples we're computing our variance on, `t` are the samples.

Reply:
I think I'm confused at `is t the absolute time when a sample is taken` or `is t the length of time interval from when the previous sample was taken`?

Reply:
This is described in the header file. The samples are times between each increment.